src/ApplicationManager.cpp

```diff
@@ -10,30 +10,30 @@
 #include "include/roboteam_ai/stp/plays/offensive/AttackingPass.h"
-#include "include/roboteam_ai/stp/plays/referee_specific/BallPlacementThem.h"
-#include "include/roboteam_ai/stp/plays/referee_specific/BallPlacementUs.h"
-#include "include/roboteam_ai/stp/plays/defensive/DefendPass.h"
-#include "include/roboteam_ai/stp/plays/defensive/DefendShot.h"
-#include "include/roboteam_ai/stp/plays/referee_specific/DefensiveStopFormation.h"
-#include "include/roboteam_ai/stp/plays/referee_specific/FreeKickThem.h"
-#include "include/roboteam_ai/stp/plays/offensive/GenericPass.h"
+//#include "include/roboteam_ai/stp/plays/referee_specific/BallPlacementThem.h"
+//#include "include/roboteam_ai/stp/plays/referee_specific/BallPlacementUs.h"
+//#include "include/roboteam_ai/stp/plays/defensive/DefendPass.h"
+//#include "include/roboteam_ai/stp/plays/defensive/DefendShot.h"
+//#include "include/roboteam_ai/stp/plays/referee_specific/DefensiveStopFormation.h"
+//#include "include/roboteam_ai/stp/plays/referee_specific/FreeKickThem.h"
+//#include "include/roboteam_ai/stp/plays/offensive/GenericPass.h"
 #include "include/roboteam_ai/stp/plays/contested/GetBallPossession.h"
-#include "include/roboteam_ai/stp/plays/contested/GetBallRisky.h"
-#include "include/roboteam_ai/stp/plays/referee_specific/Halt.h"
-#include "include/roboteam_ai/stp/plays/referee_specific/KickOffThem.h"
-#include "include/roboteam_ai/stp/plays/referee_specific/KickOffThemPrepare.h"
-#include "include/roboteam_ai/stp/plays/referee_specific/KickOffUs.h"
-#include "include/roboteam_ai/stp/plays/referee_specific/KickOffUsPrepare.h"
-#include "include/roboteam_ai/stp/plays/referee_specific/PenaltyThem.h"
-#include "include/roboteam_ai/stp/plays/referee_specific/PenaltyThemPrepare.h"
-#include "include/roboteam_ai/stp/plays/referee_specific/PenaltyUs.h"
-#include "include/roboteam_ai/stp/plays/referee_specific/PenaltyUsPrepare.h"
-#include "stp/plays/ReflectKick.h"
-#include "stp/plays/TestPlay.h"
-#include "include/roboteam_ai/stp/plays/referee_specific/TimeOut.h"
+//#include "include/roboteam_ai/stp/plays/contested/GetBallRisky.h"
+//#include "include/roboteam_ai/stp/plays/referee_specific/Halt.h"
+//#include "include/roboteam_ai/stp/plays/referee_specific/KickOffThem.h"
+//#include "include/roboteam_ai/stp/plays/referee_specific/KickOffThemPrepare.h"
+//#include "include/roboteam_ai/stp/plays/referee_specific/KickOffUs.h"
+//#include "include/roboteam_ai/stp/plays/referee_specific/KickOffUsPrepare.h"
+//#include "include/roboteam_ai/stp/plays/referee_specific/PenaltyThem.h"
+//#include "include/roboteam_ai/stp/plays/referee_specific/PenaltyThemPrepare.h"
+//#include "include/roboteam_ai/stp/plays/referee_specific/PenaltyUs.h"
+//#include "include/roboteam_ai/stp/plays/referee_specific/PenaltyUsPrepare.h"
+//#include "stp/plays/ReflectKick.h"
+//#include "stp/plays/TestPlay.h"
+//#include "include/roboteam_ai/stp/plays/referee_specific/TimeOut.h"
```

Comment:
commented; really try to document for yourself *why* you are commenting code out. Trust me, you're not going to accidentally stumble across the code later and still remember why you commented it out more than a month from now.

Reply:
This was the next step. But ill do it next times